### DISCUSSION TITLE
AccuCT with this setup?

### DISCUSSION THREAD

**Discussion by pyrodex** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#141] [Q&A] [answered]
Hello!

I am looking to switch away from IoTaWatt with their recent announcement and have a working setup with AccuCT clamps. Was trying to save money and reuse these clamps if possible. Here is their data sheet - https://www.accuenergy.com/wp-content/uploads/acuct-hinged-series-compact-split-core-current-transformer-datasheet.pdf

I have the 50A (12), 100A (2), and 200A (2) models currently in use today. I am in the US so the two 200As are on my Main feed coming into the power company and then I have the 50As spread across various circuits. I assume, based on the FAQ, and the data sheet these require me to severe the jumper since they have burden resistors right? Anyone else may have calibrated these units? Just trying to make the switch as seamless as possible on some known circuits.

**Comment by CircuitSetup** [comment, marked as answer]
I didn't realize that IoTaWatt was not being made anymore. Thanks for letting me know. It's a shame, because from what I gathered that was a great product. 

For the CTs, it looks like the AccuCT's all output 333mV when they are reading their rated peak. Since they have an internal burden resistor, you'll just need to sever the jumpers on the back of the board to disable the on-board burden resistors. 

You'll also need to calibrate the CTs to the meter using the method here: https://www.youtube.com/watch?v=BOgy6QbfeZk&t=1261s

You can start with a value of 5132 for the 50A CTs.

**Reply by pyrodex** [reply]
When you say "sever" how does one do that? With a solder iron or a pair of clippers?

**Reply by CircuitSetup** [reply]
With a knife/blade. You'd would only need to use a soldering iron if you decide to use different CTs that don't have burden resistors.

**Reply by ICPete** [reply]
Hi John,
Since I'm also interested in using alternative CTs, I'm trying to understand what will work and what won't.
I understand your circuit uses onboard 22 ohm burden resistors, which can be removed by severing their jumpers.
With the onboard resistor in place, we need a CT outputting a maximum current of 33 mA, correct?
With the onboard resistor severed, we need a CT outputting a maximum voltage of 720 mV, correct?
For reference, I notice from the AcuCT Hinged Series datasheet that their built-in burden resistors are as follows:
50A: 20R
100A: 10R
200A: 5R
Since they are all designed to output a maximum voltage of 333 mV, I conclude that their secondary max currents are as follows:
50A: 16.65 mA
100A: 33.3 mA
200A: 66.6 mA
To me that implies they all have a 3000:1 turns ratio. Just doing the calculation for fun.

Finally, if I were to locate a 200A CT that output, say, 100 mA max (i.e., 2000:1 turns ratio, which is quite common on AliExpress), could I place a resistor in parallel with the onboard 22 ohm resistor to bring the combined resistance down to, say, 7.2 ohms, in order to produce a maximum voltage of 720 mV? By my calculation that would require the added resistor to be 10.7 ohms.

John, thank you for bringing this excellent circuit to fruition, and for making it available to us!

**Comment by pyrodex** [comment]
As a test with my AccuCT’s I tried severing the JP1 pad before doing any testing and it didn’t work. I even ordered a bunch of the YHDC’s from AliExpress and they didn’t work in ct1 till soldered the jp1 back.

Test before you sever first, that was my experience. Also make sure you have a know good and repeatable load testing to calibrate as needed.

-
Richie

On Jun 18, 2023, at 1:34 PM, ICPete ***@***.***> wrote:

﻿

Hi John,
Since I'm also interested in using alternative CTs, I'm trying to understand what will work and what won't.
I understand your circuit uses onboard 22 ohm burden resistors, which can be removed by severing their jumpers.
With the onboard resistor in place, we need a CT outputting a maximum current of 33 mA, correct?
With the onboard resistor severed, we need a CT outputting a maximum voltage of 720 mV, correct?
For reference, I notice from the AcuCT Hinged Series datasheet that their built-in burden resistors are as follows:
50A: 20R
100A: 10R
200A: 5R
Since they are all designed to output a maximum voltage of 333 mV, I conclude that their secondary max currents are as follows:
50A: 16.65 mA
100A: 33.3 mA
200A: 66.6 mA
To me that implies they all have a 3000:1 turns ratio. Just doing the calculation for fun.

Finally, if I were to locate a 200A CT that output, say, 100 mA max (i.e., 2000:1 turns ratio, which is quite common on AliExpress), could I place a resistor in parallel with the onboard 22 ohm resistor to bring the combined resistance down to, say, 7.2 ohms, in order to produce a maximum voltage of 720 mV? By my calculation that would require the added resistor to be 10.7 ohms.

John, thank you for bringing this excellent circuit to fruition, and for making it available to us!

**Reply by ICPete** [reply]
Hi Richie,
So I just looked at my extra CTs from my Iotawatt system.
They don't say "333mV"; they are labeled as "50mA"!
Next I downloaded the Eagle schematic for the IotaWatt v5.0, and it clearly shows onboard 20-ohm resistors for all the CT jacks:

![image](https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/assets/136994470/3cadd23f-9b05-4cf9-a430-e0921aa8558b)

Finally, the Iotawatt specs say the input voltages are rated for 1.0V max, which makes sense if you figure 50mA * 20R = 1.0V.
So I would conclude that Iotawatt's official CTs are actually 50mA current output, not 333mV voltage output.
This means 1) you don't want to sever the jumpers for the 22R onboard resistors on the CS board, and 2) you cannot use the full specified range of the AccuEnergy CTs, as the CS system can only handle a maximum of 33mA input from each CT.

So the 50A CT should be limited to 33A max, the 100A CT should be limited to 66A max, and the 200A CT should be limited to 132A max.
Or you can probably change the 22R onboard resistor to 2/3 * 22R = 14.67R. This could be done, for example, by adding a 44R resistor in parallel with the existing 22R resistor (not sure exactly where on the board that might be feasible however!). Of course this would require recalibration afterwards...

**Reply by ICPete** [reply]
It looks to me that the easiest place to solder in an additional resistor, in parallel with the existing 22R resistor on the CS board, would be between pins 1 & 2 of the phone jack. I've circled these terminals in red in this image:
![image](https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/assets/136994470/58025714-35ed-49fe-b212-a401967edaaa)

Maybe John will comment as to whether this is a suitable solution. I'm not sure whether reducing the value of the onboard burden resistor would have any other effects, beyond accommodating a larger secondary current from a CT.

**Reply by ICPete** [reply]
From the CUI datasheet for the audio jack, the distance between terminals 1 & 2 (center-to-center) is 6.8mm. I see that a typical 1/8-watt through-hole resistor has a body length of 3.3mm, so that should fit. [A typical 1/4-watt resistor tends to be 6.3mm long, so that might be harder to form its leads to fit between the terminals.]
Of course, 44R is not a standard value, but 43R is commonly available and results in <1% error when combined with the existing 22R resistor. You're probably going to calibrate it afterwards anyway.
